README.md

```diff
@@ -100,6 +100,14 @@ continually watch Consul for updates to the catalog and perform health checks de
 external nodes it discovers. This allows externally registered services and checks to access
 the same features as if they were registered locally on Consul agents.
 
+Each ESM registers a health check for itself with the agent with
+`"DeregisterCriticalServiceAfter": "30m"`, which is currently not configurable. This means after
+failing its health check, the ESM will switch from passing status to critical status. If the ESM
```

Comment:
I think the comma after check is a comma splice.

Reply:
I was intrigued by comma rules and read up on commas after introductory phrases. It looks like it's convention to omit a comma if the introductory phrase is short and the omission doesn't lead to confusion. If the phrase is 4+ words, there is more merit to the comma. I'm leaning towards erring on the side of caution and keeping this comma in case the intro phrase is long enough for the reader to lose the flow and break of the sentence.